js/languages/en-US.json

```diff
@@ -51,13 +73,6 @@
     "provideUsername": "Please provide a username.",
     "duplicateSocialAccount": "You already have a social account of this type."
   },
-  "profileModelErrors": {
```

Comment:
Why are these deleted? These are still needed, since these are ones that can make it into the UI.